vmdb/app/presenters/tree_builder.rb

```diff
@@ -193,6 +195,8 @@ def x_get_tree_objects(options)
     object = options[:parent]
     children_or_count = case object
                         when nil                 then x_get_tree_roots(options)
+                        when ConfigurationProfileForeman then
+                          x_get_tree_cpf_kids(object, options)
```

Comment:
None of the items here are base class specific, so why is this one?  Same with ProviderForeman down below.